cli-feed-v3-3.json

```diff
@@ -0,0 +1,269 @@
+{
+  "tags": {
+    "v1": {
+      "release": "1.13.0",
+      "releaseQuality": "GA",
+      "hidden": false
+    },
+    "v1-prerelease": {
+      "release": "1.13.0",
+      "releaseQuality": "Prerelease",
+      "hidden": true
+    },
+    "v2": {
+      "release": "2.60.0",
+      "releaseQuality": "GA",
+      "hidden": false
+    },
+    "v2-prerelease": {
+      "release": "2.60.0",
+      "releaseQuality": "Prerelease",
+      "hidden": true
+    },
+    "v2-preview": {
+      "release": "2.60.0",
+      "releaseQuality": "Preview",
+      "hidden": true
+    },
+    "v3-preview": {
+      "release": "3.20.0",
+      "releaseQuality": "Preview",
+      "hidden": true
+    },
+    "v3-prerelease": {
+      "release": "3.20.0",
+      "releaseQuality": "Prerelease",
+      "hidden": true
+    },
+    "v3": {
+      "release": "3.20.0",
+      "releaseQuality": "GA",
+      "hidden": false
+    }
+  },
+  "releases": {
+    "1.13.0": {
+      "templates": "https://functionscdn.azureedge.net/public/TemplatesApi/1.0.3.10182.zip",
+      "workerRuntimeInformation": {
```

Comment:
nit: Can we do `workerRuntimes` to match `tags` and `releases` (as opposed to `tagInformation` and `releaseInformation`)